docs/lambda.asciidoc

```diff
@@ -60,6 +59,8 @@ exports.handler = apm.lambda(function handler (payload, context, callback) {
 
 The agent will now monitor the performance of your lambda function.
 
+*IMPORTANT*: During installation of the Lambda extension you'll have set a number of environmental variables for configuring the agent.  We recommend relying on these environmental variables -- while it's possible to set configuration values via the configuration object passed to `start`, doing so may interfere with the configuration required by the Lambda extension.
```

Comment:
> doing so may interfere with the configuration required by the Lambda extension.

That is just for the `ELASTIC_APM_SERVER_URL` variable, correct? Or is there other "interfere"nce I'm not aware of. I think it would be helpful to be specific.  I agree to encourage configuring via envvars, but mainly because I think it leads to a nice developer experience if/when we use a layer for the agent and automatically instrument the handler rather than needing the explicit `apm.lambda()` call.